packages/babel-runtime/scripts/build-dist.js

```diff
@@ -88,9 +115,10 @@ each(helpers.list, function (helperName) {
 
   // compat
   var helperAlias = _.kebabCase(helperName);
-  writeFile("helpers/_" + helperAlias + ".js", buildHelper(helperName));
-  writeFile("helpers/" + helperAlias + ".js", buildHelper(helperName));
+  var content = "module.exports = require(\"./" + helperName + ".js\");";
+  writeFile("helpers/_" + helperAlias + ".js", content);
+  if (helperAlias !== helperName) writeFile("helpers/" + helperAlias + ".js", content);
```

Comment:
Yes, but I mean that if you change a helper there, the babel-helpers package is published ok, but the corresponding runtime is not published (as I understand) as its dependencies are hidden in this build script and not seen by the publish script unless these build artifacts at hand are allowed to be seen by git, or the publish script is modified to track dependencies better (just for runtime), or perhaps runtime publishing is forced just occasionally.

And also a minor point, as the babel-runtime building is special in this repo (along with polyfill package, perhaps), it is a larger leap from babel-helpers/src/helpers.js to babel-runtime npm package than with any other source package here, such as from babel-helpers/src/helpers.js to babel-helpers npm package, so as a minor point checking them in could help seeing and discussing this kind of build issues faster (as it is difficult to refer to a npm package source line?). But these are just suggestions that came when I saw this nice diff of yours.